NewareNDA/NewareNDAx.py

```diff
@@ -64,8 +85,13 @@ def read_ndc(file):
     df.reset_index(drop=True, inplace=True)
 
     # Postprocessing
+    aux_df = pd.DataFrame([])
     df = df.astype(dtype=dtype_dict)
-    return df
+    if identifier[0:1] == b'\x55':

```

Comment:
Hey, probably it should be `== b'\x65'` here.

Reply:
Good catch, I'll push a fix for this.